src/store/models/coinList/currency.ts

```diff
@@ -0,0 +1,10 @@
+import "@components/UI/Dropdown/types";
+
+export const normalizeCurrency = (currencyData: string[]) => {
+  return currencyData.map((el) => {
+    return {
+      key: el,
+      value: `Market - ${el.toUpperCase()}`,
+    };
+  });
+};
```

Comment:
из стрелочных функций можно сразу возвращать значение если в теле функции нет никакой логики, такая запись более короткая
```suggestion
export const normalizeCurrency = (currencyData: string[]) =>
  currencyData.map((el) => ({
    key: el,
    value: `Market - ${el.toUpperCase()}`,
  }));

```